test/utils/ShieldUtils-test.ts

```diff
@@ -1,7 +1,27 @@
+/*
```

Comment:
I can't comment on the file, but `MegolmExportEncryption-test.ts` (renamed without changes) has a couple functions which need arg & return types defined.